src/families/cosmos/js-synchronisation.ts

```diff
@@ -112,21 +108,26 @@ const txToOps = (info: any, id: string, txs: any): Operation[] => {
 
         case "unbond":
           if (
-            attributes["amount"] &&
-            attributes["amount"].indexOf(currency.units[1].code) != -1 &&
+            attributes.amount &&
+            attributes.amount.indexOf(currency.units[1].code) != -1 &&
             attributes.validator
           ) {
             op.type = "UNDELEGATE";
             op.value = new BigNumber(fees);
             op.extra.validators.push({
-              amount: attributes["amount"].replace(currency.units[1].code, ""),
+              amount: attributes.amount.replace(currency.units[1].code, ""),
               address: attributes.validator,
             });
           }
           break;
       }
     });
 
+    if (!["IN", "OUT"].includes(op.type)) {
+      op.senders = [];
+      op.recipients = [];
+    }
+
```

Comment:
Isn't it equivalent to initializing senders and recipients to [] when creating the op? (L28 & L29)
(Unless I'm missing some case where it's not correct?)